package.json

```diff
@@ -57,7 +56,7 @@
   "scripts": {
     "build:style-plugins-screen": "node-sass assets/plugins-screen/plugins-screen.scss assets/plugins-screen/plugins-screen.css --output-style expanded && postcss -r assets/plugins-screen/plugins-screen.css",
     "build:style-plugins-screen-rtl": "rtlcss assets/plugins-screen/plugins-screen.css assets/plugins-screen/plugins-screen-rtl.css",
-    "build:webpack": "webpack",
+    "build:webpack": "calypso-build --config='./webpack.config.js'",
```

Comment:
@ockham is this change necessary? I copied directly from similar change you made in `newspack-blocks`, but here the build seems to work with or w/out this change.

Reply:
Without the `--config` option, `calypso-build` will use its own `webpack.config.js` -- which in this case means it'll use an entry point (default is `src/index`) that's different from the ones this project's `webpack.config.js` is specifying.